manager/aica_django/converters/Knowledge.py

```diff
@@ -1105,13 +1167,22 @@ def suricata_alert_to_knowledge(
         )
     )
 
+    try:
```

Comment:
What is GameMeta? Perhaps an explanatory comment here would be helpful.

Reply:
This should not be left in code, it's domain specific knowledge from Enriched Suricata. Our Suricata logs are preprocessed by another system and the GameMeta is added (for example assets information on RT/BT, which makes labeling possible)

Reply:
Okay, please go ahead and remote it.